keadm/cmd/keadm/app/cmd/debug/get.go

```diff
@@ -0,0 +1,901 @@
+		dataSource); err != nil {
+		return fmt.Errorf("Failed to register db: %v ", err)
+	}
+	orm.RegisterModel(new(dao.Meta))
+
+	// create orm
+	dbm.DBAccess = orm.NewOrm()
+	if err := dbm.DBAccess.Using(dbName); err != nil {
+		return fmt.Errorf("Using db access error %v ", err)
+	}
+	return nil
+}
+
+// IsExistName verify the filed in the resNames exists in the name
+func isExistName(resNames []string, name string) bool {
+	value := false
+	for _, v := range resNames {
+		if strings.Contains(name, v) {
+			value = true
+		}
+	}
+
+	return value
+}
+
+// Selector filter structure
+type Selector struct {
+	Key   string
+	Value string
+	Exist bool
+}
+
+// SplitSelectorParameters Split selector args (flag: -l)
+func SplitSelectorParameters(args string) ([]Selector, error) {
+	var results = make([]Selector, 0)
+	var sel Selector
+	labels := strings.Split(args, ",")
+	for _, label := range labels {
+		if strings.Contains(label, "==") {
+			labs := strings.Split(label, "==")
+			if len(labs) != 2 {
+				return nil, fmt.Errorf("Arguments in selector form may not have more than one \"==\". ")
+			}
+			sel.Key = labs[0]
+			sel.Value = labs[1]
+			sel.Exist = true
+			results = append(results, sel)
+			continue
+		}
+		if strings.Contains(label, "!=") {
+			labs := strings.Split(label, "!=")
+			if len(labs) != 2 {
+				return nil, fmt.Errorf("Arguments in selector form may not have more than one \"!=\". ")
+			}
+			sel.Key = labs[0]
+			sel.Value = labs[1]
+			sel.Exist = false
+			results = append(results, sel)
+			continue
+		}
+		if strings.Contains(label, "=") {
+			labs := strings.Split(label, "=")
+			if len(labs) != 2 {
+				return nil, fmt.Errorf("Arguments in selector may not have more than one \"=\". ")
+			}
+			sel.Key = labs[0]
+			sel.Value = labs[1]
+			sel.Exist = true
+			results = append(results, sel)
+		}
+	}
+	return results, nil
+}
+
+func HumanReadablePrint(results []dao.Meta, printer printers.ResourcePrinter, out io.Writer) error {
+	res, err := ParseMetaToAPIList(results)
+	if err != nil {
+		klog.Fatal(err)
+	}
+	for _, r := range res {
+		table, err := ConvertDataToTable(r)
+		if err != nil {
+			klog.Fatal(err)
+		}
+		if err := printer.PrintObj(table, out); err != nil {
+			klog.Fatal(err)
+		}
+		if _, err := fmt.Fprintln(out); err != nil {
+			klog.Fatal(err)
+		}
+	}
+	return nil
+}
+
+// xParseMetaToAPIList Convert the data to the corresponding list type according to the apiserver usage type
+// Only use this type definition to get the table header processing handle,
+// and automatically obtain the ColumnDefinitions of the table according to the type
+// Only used by HumanReadablePrint.
+func ParseMetaToAPIList(metas []dao.Meta) (res []runtime.Object, err error) {
+	var (
+		podList       api.PodList
+		serviceList   api.ServiceList
+		secretList    api.SecretList
+		configMapList api.ConfigMapList
+		endPointsList api.EndpointsList
+		nodeList      api.NodeList
+	)
+	value := make(map[string]interface{})
+
```

Comment:
@Rachel-Shao plz learn how to serialization and deSerialization and refactoring this func later : ). it is also an important point for implementing list-watch proxy.